run_cube.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env pybricks-micropython
+
+from pybricks.hubs import EV3Brick
+from pybricks.ev3devices import (Motor, TouchSensor, ColorSensor,
+                                 InfraredSensor, UltrasonicSensor, GyroSensor)
+from pybricks.parameters import Port, Stop, Direction, Button, Color
+from pybricks.tools import wait, StopWatch, DataLog
+from pybricks.robotics import DriveBase
+from pybricks.media.ev3dev import SoundFile, ImageFile
+import CrabRobotLibrary
+
+def run_table_1(crabot):
+	#This pushes the innovation project out(M01).
+	crabot.gyro_reset()
```

Comment:
Before resetting the gyro, we should make the robot drive backward so that we can ensure it is completely backed up against the wall/jig.